### PR TITLE
add heading and link to gist

### PR DESCRIPTION
**Pull request by mhpjay422**
This is an addition to the README for the CSV exploration. I have added a heading and a link to a github gist on the README.

The purpose of the gist is to layout notes on working on projects with CSV files and my reflections after working with them. What I should have and could have done better.

Let me know what you think and if any adjustments should be made. 

:smiley:

### CODE REVIEW

File: csv_exploration_lesson/readme.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ I started these notes after writing down to [A Digression to Turing's "CSV Explo
 
 ----------------------
 
+## Notes/gotchas from others who have worked through this guide 
+
+- [Jason Gong's notes on the lesson] (https://gist.github.com/mhpjay422/4fdfb650910b18e82ab7d1731181edf3)
+
```
Comment (josh-works) [inline csv_exploration_lesson/readme.md:18]: this is perfect! Could you add a line break after this section? 

A `-----------` after your entry.

(As others work through this in the future, I want them to add _their_ notes, and have their notes delineated from the rest of the doc)

**Review by josh-works — CHANGES_REQUESTED** [review]
A minor request/addition, then this is good to go!

**Review by josh-works — APPROVED** [review]
hero! Thanks!

**Comment by mhpjay422** [comment]
I read that making another comment opens the option to open the PR again?

**Pull request merged**